Tone/signal/Pow.ts

```diff
@@ -0,0 +1,85 @@
+import { Signal, SignalOptions } from "./Signal";
+import { WaveShaper, WaveShaperMappingFn } from "./WaveShaper";
+import { optionsFromArguments } from "Tone";
+
+/**
+ *  @class Pow applies an exponent to the incoming signal. The incoming signal
+ *         must be AudioRange.
+ *
+ *  @constructor
+ *  @param exp The exponent to apply to the incoming signal, must be at least 2. 
+ *  @example
+ * var pow = new Pow(2);
+ * var sig = new Signal(0.5).connect(pow);
+ * //output of pow is 0.25. 
+ */
+export class Pow extends Signal<number> {
+	
+	readonly name: string = "Pow";
+
+	/**
+	 * Indicates if the value should be overridden on connection
+	 */
+	readonly override = false;
+	
+	/**
+	 * The exponent
+	 */
+	private _exponent!: number;
+	
+	private _exponentScaler = new WaveShaper({
+		context: this.context,
+		mapping: this._expFunc(this._exponent),
+		length: 8192,
+	});
+
+	input = this._exponentScaler;
+
+	output = this._exponentScaler;
+
+
+	/**
+	 * @param value Constant exponent value to use
+	 */
+	constructor(value?: number);
+	constructor(options?: Partial<SignalOptions<number>>);
+	constructor() {
+		super(Object.assign(optionsFromArguments(Pow.getDefaults(), arguments, ["value"])));
+
+		this.exponent = this.value;
```

Comment:
Here you can get the passed in value by using something like

```typescript
this.value = options.value
```

Also change the variable from `exponent` to `value` to be more consistent with the previous implementation. 

This will fix your NaN issue since it'll be setting this.value to the default options.value or the passed in value.